public/js/index.js

```diff
@@ -3,13 +3,6 @@
 
 (function () {
```

Comment:
You should pass in a global `app` object to your IIFE function, so that you can attach any methods you want to be available globally to it.